packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -38,26 +38,38 @@ library ChainAssertions {
         ResourceMetering.ResourceConfig memory dflt = Constants.DEFAULT_RESOURCE_CONFIG();
```

Comment:
Can you make `vm` a private variable to this library so we don't have to pass it in everywhere?